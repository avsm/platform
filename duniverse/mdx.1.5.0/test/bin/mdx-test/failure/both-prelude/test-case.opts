--prelude=a --prelude-str=b
