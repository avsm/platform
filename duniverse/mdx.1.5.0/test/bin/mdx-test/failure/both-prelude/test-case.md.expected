only one of --prelude or --prelude-str shoud be used
